pkg/boot/boot.go

```diff
@@ -21,21 +21,17 @@ import (
 	"context"
 	"sync"
 	"time"
-)
-
-import (
-	"github.com/pkg/errors"
-
-	"go.uber.org/multierr"
-)
 
-import (
 	"github.com/arana-db/arana/pkg/config"
 	"github.com/arana-db/arana/pkg/proto/rule"
 	"github.com/arana-db/arana/pkg/runtime"
 	"github.com/arana-db/arana/pkg/runtime/namespace"
+	"github.com/pkg/errors"
+	"go.uber.org/multierr"
```

Comment:
reformat imports.

Reply:
refer to imports-formatter at https://github.com/dubbogo/tools